spec/services/cleaners/advocate_claim_cleaner_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'rails_helper'
+require 'services/cleaners/cleaner_shared_examples'
+
+RSpec.shared_examples 'delete fixed fees' do
+  it { expect { call_cleaner }.to change { claim.fixed_fees.size }.to 0 }
+end
+
+RSpec.shared_examples 'does not delete fixed fees' do
+  it { expect { call_cleaner }.not_to(change { claim.fixed_fees.size }) }
+end
+
+RSpec.shared_examples 'does not delete misc fees' do
+  it { expect { call_cleaner }.not_to(change { claim.misc_fees.size }) }
+end
+
+RSpec.shared_examples 'clear basic fees' do
+  it { expect { call_cleaner }.not_to(change { claim.basic_fees.size }) }
+  it { expect { call_cleaner }.to change { claim.basic_fees.flat_map(&:dates_attended).size }.to 0 }
+end
+
+RSpec.shared_examples 'does not clear basic fees' do
+  it { expect { call_cleaner }.not_to(change { claim.basic_fees.size }) }
```

Comment:
would `size` change since we are only clearing the basic fees, not destroying them?

Withstanding that:
```suggestion
  it { expect { call_cleaner }.not_to change { claim.basic_fees.size } }
```